shell/packages/sandstorm-ui-grainlist/grainlist.html

```diff
@@ -4,7 +4,7 @@
   <div class="grain-list">
     <div class="top-row">
     <div class="non-usage-info">
-    <h1>Grains</h1>
+    <h1>Grains <a href="#" class="question-mark"></a></h1>
```

Comment:
I agree with you. Can I do custom styles on buttons, like this icon? If so,
great; I vaguely recall that didn't actually work, but I'll test it now.​